README.md

```diff
@@ -97,3 +97,22 @@ IPSEC_FORCEUDP=
 
 If usage of keys and certificates instead of pre shared keys should be used, the code of the repo has to be extended.
 
+### VTI Interface
+
+Related environment variables:
+* IPSEC_LOCALIP
+* IPSEC_REMOTEIP
+* IPSEC_VTI_KEY
+* IPSEC_VTI_STATICROUTES
+
+If the entrypoint is provided the argument `init`, an initialisation-container is started that can create a VTI tunnel to route
+the IPSEC traffic over.
+
+To create a VTI interface, set the environment variable `IPSEC_VTI_KEY` to an integer.
+
+A VTI tunnel interface is then created with `IPSEC_LOCALIP` and `IPSEC_REMOTEIP` as endpoints and `IPSEC_VTI_KEY` as key.
```

Comment:
is that still / really necessary to have the peer IPs on the vti?

Reply:
Test make it look like no: 

```
14: vti42@NONE: <NOARP,UP,LOWER_UP> mtu 1332 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1
    link/ipip 92.43.50.69 brd 0.0.0.0
```

```
vti42: ip/ip remote any local 92.43.50.69 ttl inherit key 42
RX: Packets    Bytes        Errors CsumErrs OutOfSeq Mcasts
    122        10248        0      0        0        0
TX: Packets    Bytes        Errors DeadLoop NoRoute  NoBufs
    122        19764        0      0        0        0
```

I removed it.